### PR TITLE
Add CSV export

### PR DESCRIPTION
**Pull request by TimvdLippe**
CSV export for administrators

### CODE REVIEW

**Comment by royklip** [comment]
Beautiful

**Review by dvantetering — APPROVED** [review]
Looks good, I'll approve

**Review by royklip — APPROVED** [review]

File: karim.html [hunk 1]
```diff
@@ -234,6 +239,11 @@
 
     <label>Update all invoices <input type="submit" id="invoices" /></label>
     <template is="dom-bind" id="app">
+      <div>
+        <textarea>[[csvNames]]</textarea>
```
Comment (martijnjanssen) [inline karim.html:243]: Maybe set is to `disabled`?

Comment (TimvdLippe) [reply karim.html:243]: Um. You can select the text then I thought? Lets merge this one now and we can update later. The committee needs it now :joy:

**Pull request merged**